wm/desk.go

```diff
@@ -32,17 +32,24 @@ type x11WM struct {
 	moveResizingType  moveResizeType
 	altTabList        []desktop.Window
 	altTabIndex       int
-	screens           []*desktop.Head
-	active            *desktop.Head
-	primary           *desktop.Head
+	screens           []*desktop.Screen
+	active            *desktop.Screen
+	primary           *desktop.Screen
 	scale             float32
 
 	allowedActions []string
 	supportedHints []string
+
+	screensProvider desktop.ScreenList
```

Comment:
This should not be needed - the screenlist is handled by the desktop code in the root package - this seems to only be set in the constructor and then passed out again - so no state in the x11WM type needed...

Reply:
Fixed.